api/app/signals/apps/signals/models/history.py

```diff
@@ -54,7 +54,9 @@ def get_action(self):  # noqa: C901
             extra = self.extra or 'Verantwoordelijke afdeling (routering)'
             return f'Routering: afdeling/afdelingen gewijzigd naar: {extra}'
         elif self.what == 'UPDATE_USER_ASSIGNMENT':
-            return f'Melding toewijzing gewijzigd naar: {self.extra}'
+            if self.extra:
```

Comment:
Also add this to https://github.com/Amsterdam/signals/blob/master/api/app/signals/apps/history/models.py#L121